src/utils/material.js

```diff
@@ -10,16 +10,16 @@ module.exports.updateMap = function (shader, data) {
   var el = shader.el;
   var src = data.src;
   var material = shader.material;
-  var materialSystem = el.sceneEl.systems.material;
+  var textureSystem = el.sceneEl.systems.texture;
```

Comment:
It's a bit more consistent to use systems when we want components to access some sort of global scope. Also would be nice to inspect the cache ```sceneEl.systems.texture.cache```.